docs/docs/spec.md

```diff
@@ -81,6 +24,52 @@ This document also links to other documents. Those documents are non-normative,
 except for when we link to the Java Language Specification to defer to its
 rules.
 
+> ### The word "nullable"
+>
+> In this doc, we aim not to refer to whether a type "is nullable." Instead, we
+> draw some distinctions, creating at least four kinds of "Is it nullable?"
+> questions we can ask for any given type usage:
+>
+> 1.  What is the [augmented type] of that type usage?
+> 2.  Do I have to handle the case where `null` comes out of it?
+> 3.  Do I have to prevent `null` from going into it?
+> 4.  Is this type a subtype of that type with respect to nullness?
+>
+> ### The scope of this spec
+>
+> Currently, this spec does not address *when* tools must apply any part of the
+> spec. For example, it does not state when tools must check that the
+> [subtyping] relation holds.
+>
+> We anticipate that tools will typically apply parts of this spec in the same
+> cases that they (or `javac`) already apply the corresponding parts of the Java
```

Comment:
```suggestion
> We anticipate that tools will typically apply each part of this spec in the same
> cases that they (or `javac`) already apply the corresponding part of the Java
```

I think this is stronger. Otherwise it sounds like they'll just randomly pick parts of the spec to apply.